app/views/api/fhir/questionaires/_show.json.jbuilder

```diff
@@ -20,23 +20,28 @@ json.item do
       end
     end
     json.item do
-      json.array! form.section_questions.each do |fq|
-        json.linkId fq.id.to_s
-        json.text fq.question.content
-        type = fq.question.response_type.code
-        type ||= fq.response_set ? 'choice' : 'text'
+      json.array! form.section_questions.each do |sq|
+        json.linkId sq.id.to_s
+        json.text sq.question.content
+        type = sq.question.response_type.code
+        type ||= sq.response_set ? 'choice' : 'text'
         json.type type
         json.extension do
-
-          if fq.question.concepts && !fq.question.concepts.empty?
+          if sq.question.concepts && !sq.question.concepts.empty?
             json.child! do
-              json.partial! 'api/fhir/extension_tags', codes: fq.question.concepts
+              json.partial! 'api/fhir/extension_tags', codes: sq.question.concepts
+            end
+          end
+          if sq.program_var.present?
+            json.child! do
+              json.url 'https://sdp-v.services.cdc.gov/fhir/questionnaire-item-program-var'
```

Comment:
should this link have `/api/` in it, i.e. `https://sdp-v.services.cdc.gov/api/fhir/questionnaire-item-program-var` ?

Reply:
It could. I copied the URL structure from the extension that is used for the tags. So if I change it here, it would need to change it there too.